samples/boards/nxp/adsp/number_crunching/sample.yaml

```diff
@@ -0,0 +1,17 @@
+sample:
+  description: An application that demonstrates
+    how to integrate a proprietary library which
+    executes math functions on target.
+  name: Proprietary Library
+tests:
+  sample.app_dev.number_crunching:
+    integration_platforms:
```

Comment:
this is missing a `platform_allow:` entry -- we don't want to build samples on all platforms as this just kills CI, and only those boards that the sample have actually been tested on and that are advertised in the README should be there
https://docs.zephyrproject.org/latest/samples/sample_definition_and_criteria.html#definition-and-criteria

Reply:
done